lib/rules/sort-prop-types.js

```diff
@@ -98,18 +95,6 @@ module.exports = {
         return;
       }
 
-      function fix(fixer) {
```

Comment:
could we keep this as commented code, and do something like `const fix = undefined;` or something, so that all the rest of the code passing `fix` doesn't have to change?

Reply:
I thought it might be easier to make this easy to revert to restore the missing code when desired.  I tend to not like to keep large swaths of commented out code around, but I can do that if desired.

Reply:
generally i'd agree, but in this case i think it'd be preferable.